src/components/partner/dto/create-partner.dto.ts

```diff
@@ -49,6 +50,14 @@ export abstract class CreatePartner {
   @NameField({ nullable: true })
   readonly address?: string;
 
+  @Field(() => String, { nullable: true })
+  @Transform(({ value: str }) => (str ? externalUrlWithProtocol(str) : null))
+  readonly websiteUrl?: string;
```

Comment:
Did we decide against the `urls` array?

Reply:
Yes. I talked with Amy and Sheri. We decided to go with just 1 URL for each item for now and ticket the possibility of doing multiple URLs later.